packages/odyssey-react/src/components/NativeSelect/NativeSelect.module.scss

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  appearance: none;
+  width: 100%;
+  max-width: var(--MaxWidth);
+  padding-block: var(--PaddingBlock);
+  padding-inline-start: var(--PaddingInline);
+  padding-inline-end: calc(
+    var(--PaddingInline) + var(--IndicatorSize) + var(--PaddingInline)
+  );
+  border-width: var(--BorderWidth);
+  border-style: var(--BorderStyle);
+  /* stylelint-disable-next-line property-disallowed-list */
+  border-radius: var(--BorderRadius);
+  border-color: var(--BorderColor);
+  background-color: var(--BackgroundColor);
+  color: var(--TextColor);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  line-height: var(--LineHeight);
+
+  /* Remove caret in IE 11 */
+  &::-ms-expand {
+    display: none;
+  }
+
+  &[multiple] {
```

Comment:
is this something we want to support in styles too?

Reply:
addressed in 7c28f27